features/steps/tabs.py

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+
+"""
+Step implementations for paragraph-related features
+"""
+
+from behave import given, then, when
+
+from docx import Document
+from docx.text.tabstop import TabStop
+from docx.shared import Inches
+
+from helpers import test_docx
+
+
+# given ===================================================
+@given('a tab_stops having {count} tab stops')
+def given_a_tab_stops_having_count_tab_stops(context, count):
+    count_int = {
+        '0': 0,
+        '3': 1
+    }[count]
+    document = Document(test_docx('tab-stops'))
+    paragraph_format = document.paragraphs[count_int].paragraph_format
+    context.tab_stops = paragraph_format.tab_stops
+
+
+# when ====================================================
+
+@when('I add a tab_stop')
+def when_i_add_a_tab_stop(context):
+    tab_stops = context.tab_stops
+    tab = tab_stops.add_tab()
+    tab.position = Inches(1.75)
+
+
+@when('I delete a tab_stop')
+def when_i_delete_a_tab_stop(context):
+    tab_stops = context.tab_stops
+    tab = tab_stops[1]
+    tab.delete()
+
+
+@when('I call tab_stops.clear()')
+def when_I_call_tab_stops_clear(context):
+    tab_stops = context.tab_stops
+    tab_stops.clear()
+
+
+# then =====================================================
+
+@then('len(tab_stops) is {count}')
+def then_len_tab_stops_is_count(context, count):
+    tab_stops = context.tab_stops
+    len_tab_stops = len(tab_stops)
+    int_count = int(count)
+    assert len_tab_stops is int_count
```

Comment:
You can just make this `assert len(tab_stops) == int(count)`. It's shorter so quicker to understand, plus it mirrors the 'then' statement.